modules/ContactsNitroModule/tsconfig.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
We should not need to introduce a new ts config here? can we just use the tsconfig from the root project?

Reply:
You're right
I just cleaned it